cmd/karavictl/cmd/role.go

```diff
@@ -217,6 +223,30 @@ func validatePowerMaxStorageResourcePool(storageSystemDetails System, storageSys
 	return nil
 }
 
+func validatePowerScaleIsiPath(storageSystemDetails System, storageSystemID string, poolQuota PoolQuota) error {
+	endpoint := GetPowerScaleEndpoint(storageSystemDetails)
+	epURL, err := url.Parse(endpoint)
+	if err != nil {
+		return fmt.Errorf("endpoint is invalid: %+v", err)
+	}
+
+	epURL.Scheme = "https"
+	c, err := pscale.NewClientWithArgs(context.Background(), epURL.String(), storageSystemDetails.Insecure, 1, storageSystemDetails.User, "Administrators", storageSystemDetails.Password, "", "777")
+	if err != nil {
+		return fmt.Errorf("powerscale authentication failed: %+v", err)
+	}
+
+	if _, err := c.GetVolumeWithIsiPath(context.Background(), poolQuota.Pool, "", ""); err != nil {
+		return err
+	}
+
+	if int(poolQuota.Quota) < 0 {
+		return errors.New("the specified quota needs to be a positive number")
```

Comment:
Should the PowerScale accepted quota be 0GB or some other value (-1GB perhaps), just to highlight that quota limits are not enforced by CSM Auth?

Reply:
Thanks. Let's  reach out to Sean so that the documentation is updated for PowerScale